numba/targets/base.py

```diff
@@ -260,6 +257,12 @@ def load_additional_registries(self):
         Load target-specific registries.  Can be overriden by subclasses.
         """
 
+    def mangler(self, name, types):
```

Comment:
Add `mangler` method to base target context.  The CUDA target already define this.